python/mxnet/foo/model/vision.py

```diff
@@ -1,37 +1,14 @@
 from __future__ import division
 
-import argparse, time
```

Comment:
let's put this in example for now and move to foo.model after the interface is stable.